docs/common-patterns.rst

```diff
@@ -47,7 +47,7 @@ you receive the funds of the person who is now the richest.
 
         function becomeRichest() public payable {
             if (msg.value <= mostSent) revert NotEnoughEther();
-            pendingWithdrawals[richest] += msg.value;
+            pendingWithdrawals[richest] += mostSent;
```

Comment:
From the explanation above:

> In the following contract, if you are no longer the richest,
you receive the funds of the person who is now the richest.

So the old code should be correct.